selfdrive/car/tesla/teslacan.py

```diff
@@ -0,0 +1,113 @@
+import struct
+
+import common.numpy_fast as np
+from selfdrive.config import Conversions as CV
+
+
+# *** Honda specific ***
```

Comment:
Is this block still needed for Tesla port? Since its supposed to be "Honda Specific"?

Reply:
This is unused. There will be a much cleaner pull request with updated to 0.3.8.2 codebase in about two weeks. I am currently porting the code and testing.